microsoftgraph/microsoft-graph-explorer-v4
# Add history items as a batch

## Overview
Fixes #1745 

### Demo

Optional. Screenshots, `curl` examples, etc.

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* How to test this PR
Check out this branch and run it locally
Notice how page load time has improved
If you do not have enough history items, click on any sample queries a number of times to get the number of history items up
Reload the page and notice the fast page load time
